google-cloud-pubsub/lib/google/cloud/pubsub/subscriber.rb

```diff
@@ -306,6 +306,17 @@ def max_outstanding_bytes
         # @deprecated Use {#max_outstanding_bytes}.
         alias inventory_bytesize max_outstanding_bytes
 
+        ##
+        # Whether to enforce flow control at the client side only or to enforce it at both the client and
+        # the server.
```

Comment:
Can you link here to an official explanation of flow control? Maybe this one? https://cloud.google.com/pubsub/docs/pull#config

Reply:
Done